Nightly COPR builds are failing

**Issue Description**

After update of COPR to F41 and DNF5, srpm generation no longer works:
```
dnf builddep -y --skip-broken --spec rpmbuild/SPECS/389-ds-base.spec --best --allowerasing --setopt=install_weak_deps=False
Unknown argument "builddep" for command "dnf5". Add "--help" for more information about the arguments.
It could be a command provided by a plugin, try: dnf5 install 'dnf5-command(builddep)'
make: *** [/mnt/workdir-n4au9w6_/389-ds-base/.copr/Makefile:8: srpm] Error 2
```

Instead of installing all build dependencies, we should install the ones required for srpm generation.

--- a/.copr/Makefile
+++ b/.copr/Makefile
@@ -1,12 +1,8 @@
 srpm:
-	# Install git in the buildroot to correctly generate commit hash
-	dnf install -y git
-	# Generate spec file
-	make -f rpm.mk rpmroot
-	# Install build dependencies
-	dnf install -y dnf-plugins-core
-	dnf builddep -y --skip-broken --spec rpmbuild/SPECS/389-ds-base.spec --best --allowerasing --setopt=install_weak_deps=False
+	# Install dependencies to generate srpm
+	dnf install -y git rsync bzip2 rpm-build cargo rust npm make python3-argcomplete
 	cargo install cargo-license --root /usr
+
 	# chown files in current working directory to root:root
 	# because when npm is run as root, scripts are always run
 	# with the effective uid and gid of the working directory owner.

